yt_dlp/extractor/vlive.py

```diff
@@ -306,78 +308,93 @@ class VLiveChannelIE(VLiveBaseIE):
+            # although the api has changed, I leave the following lines as notes
+
+            # video_list = self._call_api(
+            #     'getChannelVideoList', 'Seq', channel_seq,
+            #     'channel list page #%d' % page_num, {
+            #         # Large values of maxNumOfRows (~300 or above) may cause
+            #         # empty responses (see [1]), e.g. this happens for [2] that
+            #         # has more than 300 videos.
+            #         # 1. https://github.com/ytdl-org/youtube-dl/issues/13830
+            #         # 2. http://channels.vlive.tv/EDBF.
+            #         'maxNumOfRows': 100,
+            #         'pageNo': page_num
+            #     }
+            # )
+
+            self.to_screen('Downloading channel list page#%d' % page_num)
             video_list = self._call_api(
-                'getChannelVideoList', 'Seq', channel_seq,
-                'channel list page #%d' % page_num, {
-                    # Large values of maxNumOfRows (~300 or above) may cause
-                    # empty responses (see [1]), e.g. this happens for [2] that
-                    # has more than 300 videos.
-                    # 1. https://github.com/ytdl-org/youtube-dl/issues/13830
-                    # 2. http://channels.vlive.tv/EDBF.
-                    'maxNumOfRows': 100,
-                    'pageNo': page_num
-                }
-            )
-
-            if not channel_name:
-                channel_name = try_get(
-                    video_list,
-                    lambda x: x['channelInfo']['channelName'],
-                    compat_str)
+                posts_path, posts_id,
+                'channel{channelName},contentType,postId,title,url', query_add=query_add)
 
             videos = try_get(
-                video_list, lambda x: x['videoList'], list)
+                video_list, lambda x: x['data'], list)
             if not videos:
                 break
 
             for video in videos:
-                video_id = video.get('videoSeq')
-                video_type = video.get('videoType')
-
-                if not video_id or not video_type:
+                if not channel_name:
+                    channel_name = try_get(video, lambda x: x['channel']['channelName'], compat_str) or ''
+                    if board_name and board_name != 'Unknown':
+                        channel_name += ' - ' + board_name
+                if video.get('contentType') != 'VIDEO':
+                    continue
+                video_id = video.get('postId')
+                if not video_id:
                     continue
                 video_id = compat_str(video_id)
+                video_title = video.get('title')
+                if not video_title:
+                    continue
+                video_title = compat_str(video_title)
+                video_url = url_or_none(video.get('url'))
+                if not video_url:
+                    continue
```

Comment:
I feel like this can be simplified

Isn't this better?
```suggestion
                video_id = str(try_get(video, lambda x: x['postId'], (int, str)))
                video_title = str_or_none(video.get('title'))
                video_url = url_or_none(video.get('url'))
                if not video_id or not video_title or not video_url:
                    continue
```